net-analyzer/gvm-libs/gvm-libs-11.0.0.ebuild

```diff
@@ -0,0 +1,92 @@
+# Copyright 1999-2020 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+CMAKE_MAKEFILE_GENERATOR="emake"
+inherit cmake flag-o-matic toolchain-funcs
+
+DESCRIPTION="Greenbone vulnerability management libraries, previously named openvas-libraries"
+HOMEPAGE="https://www.greenbone.net/en/"
+SRC_URI="https://github.com/greenbone/gvm-libs/archive/v${PV}.tar.gz -> ${P}.tar.gz"
+
+SLOT="0"
+LICENSE="GPL-2+"
+KEYWORDS="~amd64 ~x86"
+IUSE="extras ldap radius"
+
+DEPEND="
+	app-crypt/gpgme:=
+	dev-libs/hiredis
+	dev-libs/libgcrypt:=
+	dev-perl/UUID
+	net-libs/gnutls:=
+	net-libs/libssh:=
+	sys-libs/zlib
+	dev-vcs/git
+	dev-libs/glib
+	acct-user/gvm
```

Comment:
Sort please. Remember that we put unconditional deps before conditional ones, so just sort this highlighted list.